easybuild/easyconfigs/g/GATE/GATE-7.1-intel-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,32 @@
+name = 'GATE'
+version = '7.1'
+versionsuffix = '-Python-2.7.11'
+
+homepage = 'http://www.opengatecollaboration.org/'
+description = """GATE is an advanced opensource software developed by the international OpenGATE collaboration and
+ dedicated to the numerical simulations in medical imaging. It currently supports simulations of Emission Tomography
+  (Positron Emission Tomography - PET and Single Photon Emission Computed Tomography - SPECT), and Computed Tomography"""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+sources = [('%(namelower)s_v%(version)s.tar.gz', "tar xfvz %s")]
+source_urls = ['http://www.opengatecollaboration.org/sites/opengatecollaboration.org/files/']
+
+patches = [
+    'GATE-7.0_Makefile-prefix.patch',
+    'GATE-7.0_unistdh.patch',
+    'GATE-7.1_std-isnan.patch',
+]
+
+dependencies = [
+    ('Geant4', '10.01.p03'),
+    ('CLHEP', '2.2.0.8'),
+    ('ROOT', 'v5.34.34', versionsuffix),
+]
+builddependencies = [
+    ('CMake', '3.4.3'),
+]
+
+buildopts = 'LD="$CXX"'
```

Comment:
since this applies to all GATE versions, it should actually be done in the easyblock instead...

see https://github.com/hpcugent/easybuild-easyblocks/pull/855